config/configStructs/tapConfig.go

```diff
@@ -37,6 +37,12 @@ const (
 	DebugLabel                   = "debug"
 	ContainerPort                = 80
 	ContainerPortStr             = "80"
+	PcapDest                     = "dest"
+	PcapMaxSize                  = "maxSize"
+	PcapMaxTime                  = "maxTime"
+	PcapTimeInterval             = "timeInterval"
+	PcapKubeconfig               = "kubeconfig"
```

Comment:
not sure why you store local variable like `PcapDest`, `pcapKubeconfig` in the `config-map`. These are short term, local variable that can be share between `pcapDump.go` and `pcapDumpRuuner.go`.
This can be changed later on, but as-is, they keep space in the values.yaml and in the config-map. Suggest considering removing those from the config struct.

Reply:
These are not actually stored in config-map, if we remove them from here then we will get the following error when we run the pcapdump command

`2024-10-06T13:06:10+03:00 WRN config.go:179 > error="flag \"pcapdump.dest\" not found"
`